teams/vsp/teams/tools/backend/external-service-monitoring/bin/capture-slo-screenshots

```diff
@@ -82,34 +82,55 @@ options = Selenium::WebDriver::Firefox::Options.new
 
 begin
   # TODO: Create a template/skeleton profile to include here
+  # TODO: This ONLY works if you open `about:profiles` in a different profile window. Not sure why.
   default_profile = Selenium::WebDriver::Firefox::Profile.from_name PROFILE_NAME
-  if path == OSX_FIREFOX_PATH
-    default_profile.add_extension default_profile.instance_variable_get('@model') + EXTENSION_NAME
-  end
-  options.profile = default_profile
-  options.headless!
 
-  # TODO: Add capability for direct proxy in lieu of extension
-  # https://github.com/department-of-veterans-affairs/vets.gov-team/blob/master/Practice%20Areas/Engineering/Internal%20Tools.md#test-and-use-the-socks-proxy
-  driver = Selenium::WebDriver.for :firefox, options: options
+  default_profile['network.proxy.type'] = 1
+  default_profile['network.proxy.socks'] = '127.0.0.1'
+  default_profile['network.proxy.socks_port'] = 2001
+  default_profile['network.proxy.socks_remote_dns'] = true
+  default_profile['network.proxy.no_proxies_on'] = 'www.googletagmanager.com, www.google-analytics.com'
 
-  b = Watir::Browser.new driver
-  b.goto GRAFANA_BASE_URL
-  b.window.resize_to(1200, 1600)
+  options.profile = default_profile
+  # options.headless!
+
+  driver  = Selenium::WebDriver.for :firefox, options: options
+  browser = Watir::Browser.new driver
 rescue Errno::ENOENT => e
+  puts e
+  puts PROFILE_NAME
   puts '🦊 Close all instances of Firefox and try again...'
+  puts 'It may be necessary to go to \'about:profiles\''
   exit 1
 rescue Selenium::WebDriver::Error::UnknownError => e
-  puts '🧦 It\'s likely that you don\'t have SOCKS enabled...'
+  puts e
+  puts '🧦 Check SOCKS is enabled and try again...'
   exit 1
 end
 
-spinner.stop('Ready for capture!')
+spinner.success('Ready for capture!')
+
+begin
```

Comment:
why is this in a block?

Reply:
I had some error handling here, though clearly it's no longer present. 

Maybe it's related to the spinner code.... seems to work with it - I'll leave it for now for the sake of not breaking things